### PR TITLE
Feat/pop up options for tasks

### PR DESCRIPTION
**Pull request by niaYaro**
Costomize pop-up for every task to able to edit and delete cycle

### CODE REVIEW

File: js/edit-task.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+console.log('list', listOfEditButtons);
+const body = document.getElementById('body')
+function editTask(event) {
+    const id = event.target.dataset.id;
+    const editBoxArray = Array.from(listOfEditBox)
+    const currentBox = editBoxArray.find(function (item) {
+        return id === item.dataset.id
+    })
+    currentBox.classList.toggle('hidden')
+    // body.classList.toggle('inactive-background')
```
Comment (vitrovyi) [inline js/edit-task.js:13]: remove commented code

File: css/taskbox.css [hunk 2]
```diff
@@ -1,4 +1,5 @@
 .taskbox {
+    /* position: relative; */
```
[the remark this answers is not in the document]
Comment (niaYaro) [reply css/taskbox.css]: Done

File: js/edit-task.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const listOfEditButtons = document.querySelectorAll('.edit-option');
+listOfEditButtons.forEach(currentEditButton => currentEditButton.addEventListener('click', editTask))
+const listOfEditBox = document.querySelectorAll('.edit')
+console.log('list', listOfEditButtons);
+const body = document.getElementById('body')
```
[the remark this answers is not in the document]
Comment (niaYaro) [reply js/edit-task.js:4]: done

File: js/edit-task.js (hunk 1 above)
Comment (niaYaro) [reply js/edit-task.js]: done

File: js/show-options.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+optionButton.forEach(currentOptionButton => currentOptionButton.addEventListener('click', showPopUp));
+function showPopUp(event) {
+    console.log('click')
+    const currentPopUpArray = Array.from(popUpOfOptions)
+    const id = event.target.dataset.id;
+    const currentPopUp = currentPopUpArray.find(function (item){
+        return item.dataset.id == id;
+    })
+    currentPopUp.classList.toggle('hidden')
+    console.log(id)
```
[the remark this answers is not in the document]
Comment (niaYaro) [reply js/show-options.js]: done

**Review by niaYaro — COMMENTED** [review]
Fixed code according to your notes

**Review by vitrovyi — APPROVED** [review]

**Comment by vitrovyi** [comment]
@niaYaro  Good job!

**Pull request merged**